go-controller/pkg/nebulamode/nebulamode.go

```diff
@@ -234,10 +238,31 @@ func (m *NebulaMode) calculateAverageHue(hsv gocv.Mat) byte {
 	h := hsv.Rows() / 2
 	dw := (w * m.config.CentralRegionXPercent) / 100
 	dh := (h * m.config.CentralRegionYPercent) / 100
-	centralRegion := image.Rect(w-dw, h-dh, w+dw, h+dh)
-	cropped := hsv.Region(centralRegion)
-	mean := cropped.Mean()
-	return byte(math.Round(mean.Val1))
+
+	var bestScore float64
+	var bestAverageHue byte
+	for j := 0; j <= m.config.LeftRightPositions; j++ {
+		x := ((2*w - 2*dw) * j) / m.config.LeftRightPositions
+		centralRegion := image.Rect(x, h-dh, x+2*dw, h+dh)
+		cropped := hsv.Region(centralRegion)
+		mean := gocv.NewMat()
+		stdDev := gocv.NewMat()
+		gocv.MeanStdDev(cropped, &mean, &stdDev)
```

Comment:
I'm not confident about the `&`s here, as we don't have that in other gocv 3.4.0 calls.  (It appears to be conventional in opencv/gocv 4.1.0, but not back in 3.4.0.)